protocol/messenger.go

```diff
@@ -2526,6 +2526,49 @@ func (m *Messenger) ShareImageMessage(request *requests.ShareImageMessage) (*Mes
 	return response, nil
 }
 
+func (m *Messenger) SyncProfilePicture(ctx context.Context, request *requests.SyncProfilePicture) error {
```

Comment:
This functions seems to be triggered externally from the API, but profile pictures should be synced in response to 2 main user behaviours.

1. When the user first pairs and syncs their device see `func (m *Messenger) SyncDevices`
1. When the user updates their profile picture

If we keep this pattern we won't need to implement separate sync API methods for each item of sync

Reply:
Made required changes, thanks for pointing this out.